src/Configurator/GitignoreConfigurator.php

```diff
@@ -36,6 +40,11 @@ public function configure(Recipe $recipe, $vars, array $options = [])
     }
 
     public function unconfigure(Recipe $recipe, $vars)
+    {
+        $this->doUnconfigure($recipe);
+    }
+
+    private function doUnconfigure(Recipe $recipe, bool $silent = false)
```

Comment:
not sure we need another method, let's add `bool $silent = false` to unconfigure.

Reply:
I'll answer you the same thing as when we discussed about the addition of `array $options = []` to all the `configure()` methods: this one is an edge case, but if configurators need more metadata one day, it's better to choose between adding `array $options = []` in **all** `unconfigure()` methods, or just keep it as-is for consistency and use an internal flag.

For the initial feature we added the options because it was used in more than one configurator.
Here, the "silent" is only one edge case (and not used in other configurators, at least not yet), so IMO it's better to keep it as-is than update all `unconfigure()` methods.

WDYT?